CardStackController/Classes/CardStackController.swift

```diff
@@ -133,6 +133,16 @@ public class CardStackController: UIViewController {
         imageView.pinEdgesToSuperviewEdges()
     }
 
+    public override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(false)
+
+        guard !view.subviews.isEmpty else { return }
+
+        if let imageView = view.subviews[0] as? UIImageView {
+            imageView.removeFromSuperview()
+        }
```

Comment:
Not a big fan of this casting and accessing the `subviews` array, it could change in the future. It might be better to keep a reference to the snapshot and remove it here.

Reply:
Agreed, I'll push an alternate solution soon.